metadata.json

```diff
@@ -49,6 +49,9 @@
         "6",
         "7"
       ]
+    },
+    {
+      "operatingsystem": "Archlinux"
```

Comment:
We've a nodeset but that isn't finished yet. We're still experimenting with docker and archlinux + systemd. I will remove the statement for now at will let you know when the nodeset is working.